src/main/java/com/anthonymace/contacttasks/services/TasksService.java

```diff
@@ -0,0 +1,77 @@
+package com.anthonymace.contacttasks.services;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.http.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import javax.validation.constraints.NotNull;
+import java.util.Arrays;
+
+@Service
+public class TasksService {
+
+    RestTemplate tasksRequest = new RestTemplate();
+    private String apiUrl = "https://api.infusionsoft.com/crm/rest/v1/%s";
```

Comment:
Nit: for constants like this, our code style is to do the all caps "API_URL" style.

Reply:
Also, again not for this starter project, but this is something we'd probably want to make configurable so it doesn't require a code deployment to change.

Reply:
https://github.com/amaceing/contact-tasks/pull/1#discussion_r295944504